### PR TITLE
fix(job) fix upload of Windows native executable

### PR DESCRIPTION
**Pull request by apupier**
one (or many) of: echo, command interpolation and write in $GITHUB_ENV is not working on Working. Using glob pattern to upload artifact to avoid having to search for the exact version

### CODE REVIEW

File: .github/workflows/publish-release.yml [hunk 1]
```diff
@@ -40,13 +40,12 @@ jobs:
...
-      - name: 🛂 Find the version
-        run: 'echo "version=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)" >> $GITHUB_ENV'
       - name: Upload binaries to release
         uses: svenstaro/upload-release-action@v2
         with:
           repo_token: ${{ secrets.GITHUB_TOKEN }}
-          file: api/target/api-${{ env.version }}${{ matrix.file }}
+          file: api/target/api-*${{ matrix.file }}
+          file_glob: true
           asset_name: ${{ matrix.asset_name }}
```
Comment (apupier) [inline .github/workflows/publish-release.yml:49]: rereading the doc, in this case, the asset_name is ignored `file_glob: If set to true, the file argument can be a glob pattern (asset_name is ignored in this case) (Default: false)`

**Comment by Delawen** [comment]
I was a bad codeveloper and already pushed a fix. But I like this better. Let me merge :)

**Comment by Delawen** [comment]
Just removed the specific job and restored windows on the matrix.

**Review by Delawen — APPROVED** [review]

File: .github/workflows/publish-release.yml (hunk 1 above)
Comment (apupier) [reply .github/workflows/publish-release.yml]: an alternative could be to configure the output name of quarkus:
https://access.redhat.com/documentation/en-us/red_hat_build_of_quarkus/1.3/html/developing_and_compiling_your_quarkus_applications_with_apache_maven/proc-configuring-the-project-output_quarkus-maven
`quarkus.package.output-name`

**Pull request merged**